spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -42,4 +42,38 @@
       )
     end
   end
+
+  describe '#get_station' do
```

Comment:
should this be `context`?

Reply:
Oh I’ve been following this convention http://www.betterspecs.org/#describe (describe for the method under test, context for a scenario)

Reply:
I've been schooled ;)

Reply:
They’re aliases so all good either way!